src/awkward/_util.py

```diff
@@ -1187,3 +1188,30 @@ def _impl(array):
         raise ak._util.error(
             ValueError(f"{module.__name__} is not supported by to_arraylib")
         )
+
+
+def deprecate(
+    message,
+    version,
+    date=None,
+    will_be="an error",
+    category=DeprecationWarning,
+    stacklevel=2,
+):
```

Comment:
If you're just adding this, there might be a commented-out version elsewhere in the file.

Now that `deprecate` is added, all of the commented-out functions can be removed.

Reply:
Yes, noticed it after pushing ;)